demos/common/printf-stdarg.c

```diff
@@ -0,0 +1,1053 @@
+/*
+ *  Copyright 2001, 2002 Georges Menie (www.menie.org)
+ *  stdarg version contributed by Christian Ettinger
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
```

Comment:
We should maybe not use this file. In other demos I have seen the Logging_Winsim.c to be sufficient.